src/LaunchDarkly.ClientSdk/LdClient.cs

```diff
@@ -32,7 +32,7 @@ namespace LaunchDarkly.Sdk.Client
     /// <para>
     /// Normally, the SDK uses the exact context that you have specified in the <see cref="Context"/>. However,
     /// you can also tell the SDK to generate a randomized identifier and use this as the context's
-    /// <see cref="Context.Key"/>; see <see cref="ConfigurationBuilder.GenerateAnonymousKeys(bool)"/>.
+    /// <see cref="Context.FullyQualifiedKey"/>; see <see cref="ConfigurationBuilder.GenerateAnonymousKeys(bool)"/>.
```

Comment:
I think this is key also, when it is a user it will happen to also be the FullyQualifiedKey (otherwise it will be prefixed with kind, or one component in a multi-kind `org:randomid`, `org:randomid:user:givenid`)

Reply:
Yes, good catch.  Thanks for rescanning all these.